website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
+  useJSXTextNode: false,
+
+  ecmaFeatures: {
+    jsx: true,
+  },
+};
+
+const parserSettingsConfiguration = {
+  fields: [
+    'range',
+    'loc',
+    'tokens',
+    'comment',
+    'tolerant',
+    'useJSXTextNode',
+    {
+      key: 'ecmaFeatures',
+      title: 'ecmaFeatures',
+      fields: Object.keys(defaultOptions.ecmaFeatures),
+      settings:
+        settings => settings.ecmaFeatures || {...defaultOptions.ecmaFeatures},
+    },
+  ],
+};
```

Comment:
Ah, good point. Fixed in https://github.com/fkling/astexplorer/pull/242/commits/4912cfb93ffbc52a9aea7d21e336522d9ee82bdf